Shared/Samples/Display web scene from portal item/DisplayWebSceneFromPortalItemView.swift

```diff
@@ -0,0 +1,42 @@
+// Copyright 2023 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import SwiftUI
+
+struct DisplayWebSceneFromPortalItemView: View {
+    /// A scene of Geneva, Switzerland created from a portal item.
+    @State private var scene = {
+        // Create a portal item using a portal and an ID.
+        let portalItem = PortalItem(
+            portal: .arcGISOnline(connection: .anonymous),
+            id: .genevaSwitzerlandScene
+        )
+        
+        // Create a scene using the portal item.
+        return Scene(item: portalItem)
+    }()
+    
+    var body: some View {
+        // Create a scene view with the scene.
+        SceneView(scene: scene)
+    }
+}
+
+private extension PortalItem.ID {
+    /// The ID for the "Geneva, Switzerland Scene" portal item on ArcGIS Online.
+    static var genevaSwitzerlandScene: PortalItem.ID {
+        .init("c6f90b19164c4283884361005faea852")!
+    }
+}
```

Comment:
I would just create this ID on line `24`. We aren't getting much out of it as a computed property since we are only using it in one spot.

Reply:
We have been doing this in the samples to make them more readable. This is from the samples contribution guide:
> If it is a constant string, a primitive POD type, a portal item ID, etc., use a computed property on their respective types to save memory and make the code more readable.

Reply:
I think that would all apply if you were using the value more than once. But if that is a guideline then this is fine.